engine.fhir/src/test/java/org/opencds/cqf/cql/engine/fhir/data/TestFhirDataProviderDstu3.java

```diff
@@ -35,7 +35,7 @@ public void testFhirClient() {
 
     // @Test
     public void testDataProviderRetrieve() {
-        Dstu3FhirModelResolver modelResolver = new Dstu3FhirModelResolver();
+        Dstu3FhirModelResolver modelResolver = new Dstu3FhirModelResolver(fhirContext);
```

Comment:
Same issue for all of these. Due to some nuanced FhirContext behavior ModelResolvers need their own.